include/Enums.h

```diff
@@ -163,22 +164,22 @@ enum TurbulenceModel {
   SMAGORINSKY = 2,
   WALE = 3,
   SST = 4,
-  SST_DES = 5,
-  SST_AMS = 6,
-  SST_IDDES = 7,
+  SSTLR = 5,
+  SST_DES = 6,
+  SST_AMS = 7,
+  SSTLR_AMS = 8,
+  SST_IDDES = 9,
+  KE = 10,
+  KE_AMS = 11,
+  KO = 12,
```

Comment:
You think we need to change the references to `KW`?

Reply:
I am not sure about this one... k-omega and k-w. If you feel strongly about KW, I will make the change.